libmscore/stafftype.h

```diff
@@ -79,7 +79,7 @@ class XmlWriter;
 //---------------------------------------------------------
 
 #define NUM_OF_DIGITFRETS                 25    // the max fret number which can be rendered with numbers
-#define NUM_OF_LETTERFRETS                17    // the max fret number which can be rendered with letters
+#define NUM_OF_LETTERFRETS                38    // the max fret number which can be rendered with letters
```

Comment:
If this Shamisen extension is merged into the main code base, such a grat increase of the number of renderable fret letters is going to create problems, as neither the provided fonts nor the French tablature system itself support such a high number of frets.

Possibly, a separate constant for Shamisen should be added and checked as a limit when applicable.

Reply:
Thank you for having a look. More globally, what do you think about this PR? Is this the only thing that draw your attention?

Reply:
Well, this way to 'sneak' a new notation in on top of a very different, already existing, one is a bit unorthodox but, if it can be fully accommodated without code changes -- or with very minimal ones like the ones above -- and it works, then we have a win-win situation.

On the other hand, if more deep code changes are required and/or on several more files, I remain convinced that the way to go is to add a new stave type.